frontend/src/app/libs/user-profile-page/user-profile-page.component.ts

```diff
@@ -47,6 +47,7 @@ export class UserProfilePageComponent implements OnInit, OnDestroy {
 			firstName: this.profileForm.value.firstName,
 			secondName: this.profileForm.value.secondName,
 			mobilePhone: this.profileForm.value.mobilePhone,
+			...this.user,
```

Comment:
here possible bug
move `...this.user` to top of this object since it may erase all data from form